pkg/istio-agent/xds_proxy.go

```diff
@@ -141,6 +141,8 @@ func initXdsProxy(ia *Agent) (*XdsProxy, error) {
 		}
 	}
 
+	wasm.SetupPlatformSpecificDefaultKeyChain(ia.cfg.Platform)
+
```

Comment:
remove

Reply:
OK. Will change the initialization sequence.